front-end/pages/business-process/index.js

```diff
@@ -47,7 +47,7 @@ const BPIndex = () => {
           style={{
             width: '100%',
             height: '55%',
-            maxHeight: 'calc(55vh - 36px)',
+            maxHeight: 'calc(55vh - 37px)',
```

Comment:
This change may be a conflict with Aishwarya's work in the future. Stay tuned and revert this file (re-make the fix) when needed.